### PR TITLE
Ak pseudobulk

### PR DESCRIPTION
**Pull request by Acribbs**

### CODE REVIEW

**Comment by Acribbs** [comment]
@AlinaKurjan All checks have passed and you can review the code and then merge once you have OK'd the code

**Comment by Acribbs** [comment]
Docs on how to review: https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews

File: cribbslab/pipeline_pseudobulk.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+"""===========================
+Pseudoalignment pipeline for bulkRNAseq data
+author: Alina Kurjan
+===========================
+To run locally: python pipeline_pseudobulk.py make full --local
```
Comment (AlinaKurjan) [inline cribbslab/pipeline_pseudobulk.py:5]: you have added somehting that isnt great I think it should be changed to this..

**Review by AlinaKurjan — APPROVED** [review]

File: cribbslab/pipeline_pseudobulk.py (hunk 1 above)
Comment (Acribbs) [reply cribbslab/pipeline_pseudobulk.py:5]: your wrong

**Pull request merged**